src/patternfly/components/Select/examples/Select.md

```diff
@@ -100,6 +100,12 @@ The single select should be used when the user is selecting an option from a lis
 {{#> select select-toggle--type="div" id="select-single-typeahead-expanded-selected" select--ItemIsSelected="true" select--IsExpanded="true" select--IsTypeahead="true" select-toggle--type="div" select-typeahead--Placeholder="New Mexico"}}
 {{/select}}
 ```
+
+```hbs title=Disabled-with-typeahead
+{{#> select select-toggle--type="div" id="select-single-typeahead-disabled" select-toggle--attribute="disabled" select--IsTypeahead="true" select-typeahead--Placeholder="Choose a state"}}
```

Comment:
When `.pf-c-select__toggle` is a `<div>` (instead of a `<button>`, which it normally is), instead of using the `disabled` attribute, you'll need to use the class `.pf-m-disabled`, since `disabled` isn't valid on a `<div>`. You can reference that in the select component's documentation by searching for "pf-m-disabled" either in your dev environment or on https://www.patternfly.org/v4/documentation/core/components/select. 

In that documentation, you'll also see

> When [.pf-m-disabled] is used, `disabled` should also be added to any form elements in `div.pf-c-select__toggle`

That means you'll need to add `disabled` to the text `<input>` and `<button>` in the toggle. 

Here's the code for the `<input>` - https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Select/select-toggle-typeahead.hbs

And for the `<button>` https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Select/select-toggle-button.hbs

One approach could be to add `select--IsDisabled="true"` in this disabled code you've added here, then in both [select-toggle-typeahead.hbs](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Select/select-toggle-typeahead.hbs) and [select-toggle-button.hbs](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Select/select-toggle-button.hbs), add conditionals to test if `select--IsDisabled` exists, and if so add `disabled` to those elements.

 Feel free to try that or any other method you think will work well! It's also worth noting you don't need to worry about adding `disabled` to [select-toggle-button.hbs](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Select/select-toggle-button.hbs) if the select is expanded, since a disabled select is not expandable.